doc/release-process.md

```diff
@@ -328,7 +341,9 @@ bitcoin.org (see below for bitcoin.org update instructions).
 
       - Archive release notes for the new version to `doc/release-notes/` (branch `master` and branch of the release)
 
-      - Create a [new GitHub release](https://github.com/bitcoin/bitcoin/releases/new) with a link to the archived release notes.
+      - Create a [new GitHub release](https://github.com/bitcoin/bitcoin/releases/new) with a link to the archived release notes
+
+      - Create a pinned meta-issue for testing the release (see [here](https://github.com/bitcoin/bitcoin/issues/15555) for an example) and provide a link to it in the release announcements where useful
```

Comment:
This is not done *after* the release, but *before*. Also, I don't like the idea of accumulating bugs of completely different kinds in a single issue. This makes it harder to track. Idk I am ~0 on this

Reply:
Changed "for testing the release" to "for testing the release candidate".